dotcom-rendering/src/web/browser/initDiscussion/init.ts

```diff
@@ -2,22 +2,19 @@ import '../webpackPublicPath';
 import { startup } from '../startup';
 import { getProps } from '../islands/getProps';
 import { doHydration } from '../islands/doHydration';
-import { getName } from '../islands/getName';
 
 async function forceHydration() {
 	try {
+		const name = 'DiscussionContainer';
+
 		// Select the Discussion island element
 		const guElement = document.querySelector<HTMLElement>(
-			'gu-island[name=DiscussionContainer]',
+			`gu-island[name=${name}]`,
```

Comment:
```suggestion
			`gu-island[name=DiscussionContainer]`,
```

Reply:
Might as well inline it as it's already called 'name'?

Reply:
Oh wait, this is what it already was?! Eh?

Reply:
We need the `name` variable to `doHydration`.

Previously we said: pick an element that has this name, then find the name of this element. This was quite redundant, required more work and was more obscure about what we’re trying to achieve. It also had an extra check if a name was missing… but this could not happen if the `querySelector` failed.

Reply:
I just meant you can inline it, there's no need for the named var as it's _already_ called 'name='

Reply:
Okay, we discussed offline and I totally missed there was a second ref to this string so the var makes sense. All good!